series-tvdb.en.yaml

```diff
@@ -1139,6 +1148,20 @@ entries:
       - season: 2
         anilist-id: 21733
 
+  - title: "Slam Dunk"
+    seasons:
+      - season: 1
+        anilist-id: 170
+      - season: 2
+        anilist-id: 170
+        start: -25
```

Comment:
Forgot that we have a feature to combine multiple seasons into one automatically. See One Piece in this mapping file.

Omitting the `start` for every season should have the same result as the current mapping.

Can you test it?

Same is true for Space Brothers and D.Gray-Man.